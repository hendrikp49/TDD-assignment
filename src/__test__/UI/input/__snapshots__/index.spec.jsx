import Input from "@/components/UI/input";
import { render } from "@testing-library/react";

describe("Input Component", () => {
  it("render input component", () => {
    const { container } = render(<Input />);
    expect(container).toMatchSnapshot();
  });
  it("render input textarea", () => {
    const { container } = render(<Input type="textarea" value="" />);
    expect(container).toMatchSnapshot();
  });
});
