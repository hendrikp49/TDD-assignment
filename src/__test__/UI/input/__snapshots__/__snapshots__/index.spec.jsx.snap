// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component render input component 1`] = `
<div>
  <input />
</div>
`;

exports[`Input Component render input textarea 1`] = `
<div>
  <textarea
    type="textarea"
  />
</div>
`;
